src/main/java/io/woo/htmltopdf/wkhtmltopdf/WkHtmlToPdfLoader.java

```diff
@@ -35,7 +35,8 @@ static WkHtmlToPdf load() {
                 throw new RuntimeException(e);
             }
         }
-        return (WkHtmlToPdf)Native.loadLibrary(libraryFile.getAbsolutePath(), WkHtmlToPdf.class);
+        System.setProperty("jna.library.path", libraryFile.getParent());
+        return Native.loadLibrary("wkhtmltox", WkHtmlToPdf.class);
```

Comment:
This will fail to load the 32-bit binaries, and will override user-specified `jna.library.path` (generally a bad idea if users of htmltopdf also use jna and depend on this setting.)
`Native#loadLibrary` should be able to load a library from path in MacOS, it's worth checking what makes it fail.

Reply:
Thinking about it, try running the same code just without the `System#setProperty` line, if it works then that means it's loading `wkhtmltox` from your `PATH` (there is no `dylib` included in `htmltopdf-java`.)